test/e2e/signature-request.spec.js

```diff
@@ -134,25 +135,33 @@ describe('MetaMask', function () {
       await delay(regularDelayMs)
 
       await waitUntilXWindowHandles(driver, 3)
-      windowHandles = await driver.getAllWindowHandles()
+      const windowHandles = await driver.getAllWindowHandles()
 
       extension = windowHandles[0]
-      popup = await switchToWindowWithTitle(driver, 'MetaMask Notification', windowHandles)
-      dapp = windowHandles.find(handle => handle !== extension && handle !== popup)
+      dapp = await switchToWindowWithTitle(driver, 'E2E Test Dapp', windowHandles)
+      popup = windowHandles.find(handle => handle !== extension && handle !== dapp)
+
+      await driver.switchTo().window(popup)
 
       await delay(regularDelayMs)
-      const approveButton = await findElement(driver, By.xpath(`//button[contains(text(), 'Connect')]`))
```

Comment:
It looks like you've removed the step where the "Connect" button on the dapp is clicked. It doesn't connect on page-load anymore, so this step is required.

Reply:
Huh, the same problem is present in the `ethereum-on` e2e tests. I must have missed that when rebasing 🤔

Reply:
See line 131 above

Reply:
Ah I see now, sorry. Those three lines in particular were missing from the diff 🤦‍♂